client/python/armada_client/event.py

```diff
@@ -0,0 +1,100 @@
+import enum
+import typing
+from dataclasses import dataclass
+
+import google.protobuf.timestamp_pb2 as timestamp_pb2
+
+from armada_client.armada import event_pb2, queue_pb2
+from armada_client.k8s.io.apimachinery.pkg.api.resource import (
+    generated_pb2 as resource_pb2,
+)
+
+
+class EventType(enum.Enum):
```

Comment:
Is this a duplication of types that are in the protobuf? Is this code likely to break as Armada's API evolves?

Reply:
Yeah. In its current form it would break, but I can add another enum for "unknown", which won't break any of its type access.